pkg/api/v1/resource/helpers.go

```diff
@@ -99,7 +106,7 @@ func PodRequests(pod *v1.Pod, opts PodResourcesOptions) v1.ResourceList {
 			containerReqs = applyNonMissing(containerReqs, opts.NonMissingContainerRequests)
 		}
 
-		if container.RestartPolicy != nil && *container.RestartPolicy == v1.ContainerRestartPolicyAlways {
```

Comment:
Should we change these as well?

```
plugin/pkg/admission/limitranger/admission.go
594:            if container.RestartPolicy != nil && *container.RestartPolicy == api.ContainerRestartPolicyAlways {
632:            if container.RestartPolicy != nil && *container.RestartPolicy == api.ContainerRestartPolicyAlways {

staging/src/k8s.io/kubectl/pkg/util/resource/resource.go
68:             if container.RestartPolicy != nil && *container.RestartPolicy == corev1.ContainerRestartPolicyAlways {
109:            if container.RestartPolicy != nil && *container.RestartPolicy == corev1.ContainerRestartPolicyAlways {
```

Reply:
Maybe not, there're only two occurrence in each file, I think it should be fine.
If we do, I tend to leave this as a follow up :)